templates/registry/markdown/attribute_namespace.md.j2

```diff
@@ -0,0 +1,43 @@
+{#- This template is rendered per top-level registry namespace. -#}
+{#- It consists of two variables: -#}
+{#- - id: The top-level namespace id. -#}
+{#- - groups: A sequence of all attribute groups under this namespace. -#}
+{#-           This includes deprecated groups. -#}
+{% import 'stability.j2' as stability %}
+{% import 'notes.j2' as notes %}
+{%- set my_file_name = ctx.id | lower | kebab_case ~ ".md" -%}
+{{- template.set_file_name(my_file_name) -}}
+
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# {{ ctx.id | title_case }}
+
+{% if ctx.groups | length > 1 %}{% for group in ctx.groups | sort(attribute="id") %}
+{%- set group_name = group.id | split_id | list | reject("eq", "registry") | join(" ") -%}
+- [{{ group_name | title_case }}](#{{group_name | kebab_case }}-attributes)
+{% endfor %}{% endif %}
+{% for group in ctx.groups | sort(attribute="id") %}
```

Comment:
see my other comment. I think we should split this into non-deprecated and deprecated attributes.
Deprecated attribute groups have the naming convention to end with `.deprecated`, we could use that to filter maybe?

Reply:
This is planned (see issue description), but I think not required for the first iteration and merge of this PR

Reply:
ahh, missed that in the description, thanks!